routes/web.php

```diff
@@ -30,3 +30,4 @@
 
 Route::post('/verifyExcelSheet', 'AdminController@checkEmailsInJson');
 Route::post('/submitGoodParticipants','ParticipantController@addGoodParticipantsToParticipantsTable');
+Route::post('/getReminder','ModuleProgressController@remindUserofModule');
```

Comment:
I would consider moving the api routes that the front end uses into the `routes/api.php` that way the CSRF token protection isn't needed to be modified. Or if the default Laravel configuration is used then we don't really need to explicitly tell Laravel to exclude certain routes.